airbyte-cdk/java/airbyte-cdk/db-sources/build.gradle

```diff
@@ -16,9 +16,11 @@ plugins {
     id "java-library"
     id "java-test-fixtures"  // https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures
 }
-// TODO: Remove this after debugging
+
 test {
+    systemProperty 'user.timezone', 'UTC'
```

Comment:
This seems like it should be set repo-wide. Can you file an issue and assign me to it, please? I gotta run.

Reply:
@postamar Probably.  Will create a ticket.

Reply:
@postamar Done:  https://github.com/airbytehq/airbyte/issues/31060

Reply:
Thanks!